### ISSUE TITLE
Typescript import issue in sql.execute.knex.ts

### ISSUE THREAD

**Issue by NickBolles** [feathers-plus/generator-feathers-plus#122]
### Steps to reproduce
Initiate a typescript project with feathers+ cli with typescript and add a service or two, then add graphql

I'm not actually using knex/sql but it generates an `sql.execute.knex.ts` file in the graphql service folder. 

### Expected behavior
Everything works. haha. no but importing Knex correctly.

### Actual behavior
A typescript compile error.
```
4:10 Module '"C:/newsync/Projects/RestaurantApp/app4/node_modules/@types/knex/index"' has no exported member 'Knex'.
    2 | // Execute raw SQL statement for GraphQL using Knex. (Can be re-generated.)
    3 | import { App } from '../../app.interface';
  > 4 | import { Knex } from 'knex';      |          ^    5 | // !code: imports // !end
    6 |
    7 | let dialects: any = {
Version: typescript 3.1.6
Time: 36463ms
```

I think this could be fixed with this: 
```
import { Client } from 'knex';
```

and 
```
  let knex = app.get('knexClient') as Client;
```
### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working): 
   Generator Version 0.6.68
   @types/knex: 0.15.1

**NodeJS version**: 9.11.2

**Operating System**: Win 10

**Typescript**: 3.1.6

**Comment by eddyystop** [comment]
Don't forget to make sure the tests run before publishing.

**Comment by NickBolles** [comment]
Sure thing! It might have to be tomorrow though.

**Comment by eddyystop** [comment]
Published as cli+ 0.7.71 which run generator v0.6.69. Thanks!
